mozilla_schema_generator/configs/glean.yaml

```
@@ -89,30 +89,59 @@ metrics:
         not:
           - glean_ping_info
       type: labeled_string
-  labeled_rate:
+
+  # Metric types affected by bug 1737656;
```

I would make this a link

```suggestion
  # Metric types affected by https://bugzilla.mozilla.org/show_bug.cgi?id=1737656
```